bookdown/03-optimization.Rmd

```diff
@@ -44,14 +45,20 @@ This sub-section will provide instructions on how to implement nested resampling
 
 ## Hyperparameter Tuning {#tuning}
 
+Hyperparameters are second-order parameters of machine learning models that, while often not explicitly optimized during the model estimation process, can have important impacts on the outcome and predictive performance of a model. 
+Typically, hyperparameters are fixed before running a model. 
+However, because the output of a model can be sensitive to the specification of hyperparameters, it is often recommended to make an informed decision about which hyperparameters may yield better model performance. 
+In many cases, hyperparameters may be chosen _a priori_, but it can be advantageous to try different sets of hyperparameters before fitting your model on the training data. 
```

Comment:
```suggestion
In many cases, hyperparameter settings may be chosen _a priori_, but it can be advantageous to try different settings before fitting your model on the training data. 
```